tests/Unit/inc/functions/getRocketSamplePermalink.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace WP_Rocket\Tests\Unit\inc\functions;
+
+use Brain\Monkey\Functions;
+use WPMedia\PHPUnit\Unit\TestCase;
+
+/**
+ * @covers ::get_rocket_sample_permalink
+ * @group Functions
+ * @group Posts
+ */
+class Test_GetRocketSamePermalink extends TestCase {
+
+	public static function setUpBeforeClass() {
+		parent::setUpBeforeClass();
+
+		require_once WP_ROCKET_PLUGIN_ROOT . 'inc/functions/posts.php';
+	}
+
+	public function tearDown() {
+		parent::tearDown();
+
+		remove_filter( 'editable_slug', [ $this, 'editable_slug_cb' ] );
```

Comment:
Stray line from copy/paste I think :)

Reply:
You're right. Removed the `tearDown()` method here https://github.com/wp-media/wp-rocket/pull/2485/commits/23fcefe8c3193109574aa7cfd8a47f28d876ca49